__tests__/src/config.test.ts

```diff
@@ -64,8 +65,8 @@ describe("Given a configuration", () => {
     process.env.GITHUB_EVENT_PATH = path.join(root, "resources", "github-event-path.json") // overseeds the previous two env vars
 
     const config = configFactory.getConfiguration()
-
     expect(config.shaArchive).toEqual("https://api.github.com/repos/mpermar/vib-action-test/tarball/a-new-branch")
```

Comment:
You could add `expect(config.shaArchive).toBeDefined()` to be consistent with other tests, although that `expect` is already sufficient.